app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -45,6 +45,10 @@ def x_get_tree_roots(count_only = false, _options)
       top_nodes << MiqProductFeature.obj_features[additional_feature][:feature]
     end
 
+    %w(all_vm_rules sui).each do |additional_feature|
```

Comment:
@chalettu you dont need to add this new block, all you need to do is add sui to the list here https://github.com/chalettu/manageiq-ui-classic/blob/bdb21eebe86effb684f421483d979a2074dd7f1f/app/presenters/tree_builder_ops_rbac_features.rb#L44